### ISSUE TITLE
USER ISSUE: Exception in NetObjectManager.GetInitialObjectStates

### ISSUE THREAD

**Issue by lemarkis** [StrangeLoopGames/EcoIssues#5496]
**Version:** 0.7.2.0 beta staging-f6bf50db

**Steps to Reproduce:**
Don't know, just walking...

**Expected behavior:**
It should'nt have popped an exception up.

**Actual behavior:**
The following error message popped up:
```
Caught exception:

<size=60,00%>La référence d'objet n'est pas définie à une instance d'un objet.


Stack:
   à Eco.Shared.Networking.NetObjectManager.GetInitialObjectStates(IEnumerable`1 newObjects, INetObjectViewer viewer, ICollection`1 monitored)
   à Eco.Plugins.Networking.Clients.Client.Update(Stopwatch timer)
   à Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_1()
</size>
```

**Comment by MarjieAVolk** [comment]
Save file that saw this: https://www.dropbox.com/s/oi316fy96wei2x3/Game.zip?dl=0

**Comment by metachronism** [comment]
should be fixed in 7.3
